packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -24,32 +24,40 @@ const { prefix } = settings;
  * @param {object} props react proptypes
  * @returns {object} JSX object
  */
-const CardLink = ({
+const Card = ({
   title,
   href,
   content,
   icon: Icon,
   image,
   className,
+  type,
   ...props
 }) => {
-  if (!title || !href) {
-    return null;
-  }
-
-  return (
+  return type === 'link' ? (
     <ClickableTile
-      data-autoid={`${stablePrefix}--card-link`}
-      className={classNames(`${prefix}--card-link`, className)}
+      data-autoid={`${stablePrefix}--card`}
+      className={classNames(`${prefix}--card`, className)}
       href={href}
       {...props}>
-      <Image {...image} classname={`${prefix}--card-link__img`} />
-      <div className={`${prefix}--card-link__wrapper`}>
-        <h3 className={`${prefix}--card-link__title`}>{title}</h3>
+      <Image {...image} classname={`${prefix}--card__img`} />
+      <div className={`${prefix}--card__wrapper`}>
+        <h3 className={`${prefix}--card__title`}>{title}</h3>
         {optionalContent(content)}
         {renderFooter(Icon)}
       </div>
     </ClickableTile>
+  ) : (
+    <Tile
+      data-autoid={`${stablePrefix}--card`}
+      className={classNames(`${prefix}--card`, className)}>
+      <Image {...image} />
```

Comment:
You added the class name above, but I don't see it down here. Another reason it might be good to extract this section out so they are consistent as we discussed yesterday.